### PR TITLE
feat: Configure tools application registry category - EXO-61480 - Meeds-io/MPIs#37

### PR DESCRIPTION
**Pull request by sofyenne**
This change is going to add the tasks application under the tools application category

### CODE REVIEW

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: task-management/src/main/webapp/WEB-INF/conf/task-addon/application-configuration.xml [hunk 1]
```diff
@@ -168,6 +168,56 @@
...
+              </collection>
+            </field>
+            <field name="applications">
+              <collection type="java.util.ArrayList">
+                <value>
+                  <object type="org.exoplatform.application.registry.Application">
+                    <field name="applicationName"><string>tasks</string></field>
+                    <field name="categoryName"><string>tools</string></field>
+                    <field name="displayName"><string>My Tasks</string></field>
+                    <field name="description"><string>Task  application</string></field>
```
Comment (azayati) [inline task-management/src/main/webapp/WEB-INF/conf/task-addon/application-configuration.xml]:
```suggestion
                    <field name="description"><string>Task application</string></field>
```
And separate the tags

**Review by azayati — CHANGES_REQUESTED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: task-management/src/main/webapp/WEB-INF/conf/task-addon/application-configuration.xml (hunk 1 above)
Comment (sofyenne) [reply task-management/src/main/webapp/WEB-INF/conf/task-addon/application-configuration.xml]: done !

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**